WooCommerce/Classes/AppDelegate.swift

```diff
@@ -263,7 +263,7 @@ private extension AppDelegate {
     ///
     func setupNoticePresenter() {
         var noticePresenter = ServiceLocator.noticePresenter
-        noticePresenter.presentingViewController = window?.rootViewController
+        noticePresenter.presentingViewController = appCoordinator?.tabBarController
```

Comment:
This means that we can't show any notice while in the login flow, is that ok?

Reply:
It seems to work also during the login flow. This is the site picker, immediately showed after the login in. I think it's the first screen where we use the notice presenter.

<img src="https://user-images.githubusercontent.com/495617/105736614-49bd9a00-5f35-11eb-971f-2d8b247d818c.png" width=300 />

Reply:
Maybe the login module is not using the global notice presenter but a one of it's own.

Reply:
Currently it seems that these view controllers are using their own notice presenter:
- `StorePickerViewController`
- `ManualTrackingViewController`
- `ShipmentProvidersViewController`
- `RefundConfirmationViewController`
- `NewNoteViewController`